### PR TITLE
Release notes for 1.5.0 (CHANGELOG)

### LINKED ISSUE #2282: Release notes for 1.5.0 (CHANGELOG)
* Color of selected tab in tablet should follow private tab color - #2260
* Restart browser is causing the brave crash after enabling rewards through onboarding modal - #2240
* Upgrade to Chromium 78.0.3904.90 - #2221
* Rewards isn't always enabled via onboarding - #2203 
* Internal Issue - #2170
* Hide Rewards Icon in Private Mode - #2156
* Ads & Trackers Blocked text update - #2149
* Notifications wont open Brave - #2138
* Implement sync changes to match Desktop folder mapping - #2135
* Private theme missing on 1.4.0 - #2078
* creating NTP automatically when last tab is closed and "Closing all tabs closes Brave" is disabled - #2031
* Hide https:// or http:// on Android URL bar - #1592
* Color of USD should be lighter - #1151
* date format on monthly tip thank you banner - #1309

### PR DESCRIPTION
**Pull request by kjozwiak**
Fix #2282

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+ - Hide Brave Rewards icon under Private Mode. ([#2156](https://github.com/brave/browser-android-tabs/issues/2156))
+ - Added "Brave License" under "Legal Information". ([#2170](https://github.com/brave/browser-android-tabs/issues/2170))
+ - Updated sync folder mapping to match desktop. ([#2135](https://github.com/brave/browser-android-tabs/issues/2135))
+ - Updated "Ads and Trackers Blocked" to "Ads & Trackers Blocked" under New Tab Page. ([#2149](https://github.com/brave/browser-android-tabs/issues/2149))
+ - Fixed notifications not working in certain cases. ([#2138](https://github.com/brave/browser-android-tabs/issues/2138))
+ - Fixed browser crash when restarting after enabling rewards through onboarding modal. ([#2240](https://github.com/brave/browser-android-tabs/issues/2240))
+ - Fixed rewards not always being enabled when opt in via the onboarding modal. ([#2203](https://github.com/brave/browser-android-tabs/issues/2203))
+ - Fixed new tab not being created when the last remaining tab is closed and "Closing all tabs closes Brave" is disabled. ([#2031](https://github.com/brave/browser-android-tabs/issues/2031))
```
Comment (srirambv) [inline CHANGELOG.md]: May be add this one as a new implementation rather than fixed. its a new feature that is added

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+ - Fixed incorrect date format being used under tipping banner for certain locales. ([#1309](https://github.com/brave/browser-android-tabs/issues/1309))
+ - Fixed incorrect color being used for balance under Brave Rewards. ([#1151](https://github.com/brave/browser-android-tabs/issues/1151))
+ - Fixed private theme missing under Private Mode. ([#2078](https://github.com/brave/browser-android-tabs/issues/2078))
```
Comment (srirambv) [inline CHANGELOG.md]: This is a new design that is added not fixing an existing one. It was not implemented initially when dark/light theme was done.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,22 @@
...
+ - Fixed private theme missing for active tabs in Private Mode when using tablet. ([#2260](https://github.com/brave/browser-android-tabs/issues/2260))
```
Comment (srirambv) [inline CHANGELOG.md]: This wasn't shipped and then fixed so needs to be removed from the release notes.

**Comment by kjozwiak** [comment]
@srirambv @SergeyZhukovsky pushed changes, please take a look 👍

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+ - Added private theme when using Private Mode. ([#2078](https://github.com/brave/browser-android-tabs/issues/2078))
+ - Added "Brave License" under "Legal Information". ([#2170](https://github.com/brave/browser-android-tabs/issues/2170))
+ - Updated sync folder mapping to match desktop. ([#2135](https://github.com/brave/browser-android-tabs/issues/2135))
+ - Updated "Ads and Trackers Blocked" to "Ads & Trackers Blocked" under New Tab Page. ([#2149](https://github.com/brave/browser-android-tabs/issues/2149))
+ - Fixed notifications not working in certain cases. ([#2138](https://github.com/brave/browser-android-tabs/issues/2138))
+ - Fixed browser crash when restarting after enabling rewards through onboarding modal. ([#2240](https://github.com/brave/browser-android-tabs/issues/2240))
+ - Fixed rewards not always being enabled when opt in via the onboarding modal. ([#2203](https://github.com/brave/browser-android-tabs/issues/2203))
+ - Fixed new tab not being created when the last remaining tab is closed and "Closing all tabs closes Brave" is disabled. ([#2031](https://github.com/brave/browser-android-tabs/issues/2031))
```
Comment (srirambv) [inline CHANGELOG.md]: Looks like this wasn't removed after adding it on line 7

Comment (kjozwiak) [reply CHANGELOG.md]: Good catch 👍

**Comment by kjozwiak** [comment]
@srirambv removed the duplicate item that wasn't removed 👍

**Pull request merged**